processors/HNL.py

```diff
@@ -203,6 +194,65 @@
        leptonCollection = lambda event: event.tightMuon[0] 
      )
 )
+
+analyzerChain.append(
+    JetMetUncertainties(
+	era = globalOptions["year"],
+	globalTag=jecTags[globalOptions["year"]],
+	jerTag = jerTags[globalOptions["year"]],
+	isData= args.isData 
+    )
+)
+
+
+if isMC:
+	
+	for systName,collection in [
+        	("nominal",lambda event: event.jets_nominal),
+	        ("jerUp",lambda event: event.jets_jerUp),
+	        ("jerDown",lambda event: event.jets_jerDown),
+	        ("jesTotalUp",lambda event: event.jets_jesUp["Total"]),
+	        ("jesTotalDown",lambda event: event.jets_jesDown["Total"]),
+    	]:
+
+        	analyzerChain.append(
+            		JetSelection(
+                		inputCollection=collection,
+				leptonCollection=lambda event: event.tightMuon,		
+	                	outputName="selectedJets_"+systName,
+		                storeKinematics=['pt','eta'],
+        			globalOptions=globalOptions
+			)
+	        )
+
+	
+		analyzerChain.append(
+			EventObservables(
+			jetCollection = collection,
+			leptonCollection = lambda event: event.tightMuon[0],
+			outputName="Jet_Muon_"+systName
```

Comment:
We should change this name, I put it like that but it's a bit stupid. Maybe "EventObservables_"+systName

Reply:
this is the output name for the event observables. 
outputName = "Jet_Muon", 
So i would like to keep the same module output +systName.

Reply:
I'd suggest changing the default name in EventObservables.py to "EventObservables" too

Reply:
it was like that before , and it was who changed it to Jet_Muon....

Reply:
As I said in my first comment, that was a bit stupid..

Reply:
i don't really see what is your issue here, kind of NNNNLO detail. But Ok i will call it as you want..